nubis/puppet/files/nubis_users/nubis-users.tmpl

```diff
@@ -0,0 +1,27 @@
+# Generated by confd {{ datetime }}
+---
+nubis_users::user:
+  # global-admins{{ range $dir := lsdir "/global-admins" }}{{ if exists (print "/global-admins/" $dir "/sshPublicKey") }}
```

Comment:
Actually this isn't a commented out piece of code its, `#global-admins` is actually a comment to indicate in the yaml file that this chunk of code is for `global-admins` and the  `{{ range }}` line is on the same line as the comment is so that we can get a neater output without extra lines